corehq/apps/tour/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+import json_field
+
+
+class GuidedTours(models.Model):
+    web_user = models.CharField(max_length=80, null=False)
+    tours = json_field.JSONField(
+        default={},
+    )
+    history = json_field.JSONField(
```

Comment:
maybe we want to track what tours they've taken, but I suppose that information could also live in `tours`